05_iigo/index.tex

```diff
@@ -429,51 +452,522 @@ \section{IIGO Session order}
+\subsubsection{Sanction Score}
+The judicial branch is able to evaluate all the actions that have been taken by the islands in the archipelago. 
+The holder of the Judge role is given the choice to upload custom scores for any rules which they would like to penalise more or less heavily if broken.
+Any custom scores are then broadcast to all islands, to give them more or less incentive to break these rules. \\
+
+Once the Judge has inspected the events of the past turn and whether or not islands have broken any rules, the judicial branch consequently uses the Judge's custom scoring to calculate the sanction score of each island.
+
+\subsubsection{Sanction Tier}
+With the combined sanction score of each island calculated, the Judge must decide which sanction tier to assign each island. 
+There are 5 sanction tiers as well as a \emph{No sanction} option. The Judge can upload custom \emph{Sanction score} thresholds for each tier, or they can choose to stick with default values.
+The judicial branch uses either the custom thresholds or default values, as instructed, and considers the sanction scores of each island to place them in a sanction tier. \\
+The sanction tier that each island falls in is broadcasted to all islands.
+
+Each sanction tier has an associated rule:
 \begin{itemize}
-    \item \textbf{Diplomatic sanctions}: Although having the potential of being a good alternative for severer sanctions discussed in  Section~\ref{sec:sanctions}, diplomatic sanctions are \emph{not} implemented within the scope of the coursework. \\
+    \item Sanction tier 1: \begin{rule_IIGO} 
+        An island in sanction tier $1$ must pay a sanction of $0$\% of their current personal resources plus a constant amount of $10$.
+    \end{rule_IIGO}
+    \item Sanction tier 2: \begin{rule_IIGO} 
+        An island in sanction tier $2$ must pay a sanction of $20$\% of their current personal resources plus a constant amount of $10$.
+    \end{rule_IIGO}
+    \item Sanction tier 3: \begin{rule_IIGO} 
+        An island in sanction tier $3$ must pay a sanction of $30$\% of their current personal resources plus a constant amount of $10$.
+    \end{rule_IIGO}
+    \item Sanction tier 4: \begin{rule_IIGO} 
+        An island in sanction tier $4$ must pay a sanction of $50$\% of their current personal resources plus a constant amount of $10$.
+    \end{rule_IIGO}
+    \item Sanction tier 5: \begin{rule_IIGO} 
+        An island in sanction tier $5$ must pay a sanction of $80$\% of their current personal resources plus a constant amount of $10$.
+    \end{rule_IIGO}
+\end{itemize}
+
+These rules are used to calculate the exact resource payment that will be required of each island to pay off their sanction. These specific payments are only broadcasted to the islands that must pay them.
```

Comment:
Add a note saying an island can have multiple sanctions, since each sanction has a duration? I think this is how this works last time I spoke to infra.

Reply:
thanks, doing it now

Reply:
adding a footnote now.